pybossa/view/projects.py

```diff
@@ -1502,9 +1502,11 @@ def tasks_browse(short_name, page=1, records_per_page=None):
     try:
         args = parse_tasks_browse_args(request.args)
         view_type = request.args.get('view')
+        task_browse_default_page_size = 10
+        task_list_default_page_size = 30
```

Comment:
Minor. At once, it seems they're total number of default page count. As they are default records per page, does it make sense to term them as
```python
taskbrowse_default_records_per_page = 10
tasklist_default_records_per_page = 30
```